modules/bullet/shape_bullet.cpp

```diff
@@ -113,6 +113,10 @@ btCapsuleShapeZ *ShapeBullet::create_shape_capsule(btScalar radius, btScalar hei
 	return bulletnew(btCapsuleShapeZ(radius, height));
 }
 
+btCylinderShape *ShapeBullet::create_shape_cylinder(btScalar radius, btScalar height) {
+	return bulletnew(btCylinderShape(btVector3(radius, height / btScalar(2), radius)));
```

Comment:
is not required to use `btScalar(2)`, you can just use 2.0

Reply:
Ok now it doesn't crash and the user gets a chance to change the physics engine setting in editor

Right now there is a bit of console spam when loading scenes with cylinder collisions nodes and when resizing new cylinder resources

An alternative would be to implement a dummy ShapeSW which does nothing or falls back to a convex collider but dunno... could introduce more problems than it solves

Reply:
It's enough if you show a warning only when the cylinder is created

Reply:
Sorry, to be clear, this is what the user sees when he creates a cylinder collision shape in GodotPhysics:

![cyli_err](https://user-images.githubusercontent.com/10036337/41510575-73cd2260-7267-11e8-8da7-502e3ad8bf37.png)

As you can see the console outputs the main error message, but also some failed assertions.
Same happens when loading a scene that contains cylinder collision shapes (slightly more asserts tbh)

Is it ok that way?